packages/augur-ui/src/modules/portfolio/components/common/box-header.tsx

```diff
@@ -27,22 +27,14 @@ export interface BoxHeaderProps {
 const BoxHeader = (props: BoxHeaderProps) => (
   <>
     <div className={Styles.ShowOnMobile}>
-      <div>
-        <div className={Styles.RightContent}>{props.rightContent}</div>
-        {props.bottomRightBarContent && (
-          <div className={Styles.BottomRightContent}>
-            {props.bottomRightBarContent}
-          </div>
-        )}
-      </div>
       <div
         className={classNames(Styles.Middle, {
           [Styles.isSwitched]: props.switchHeaders,
         })}
       >
         {props.bottomBarContent && (
           <div
-            className={classNames(Styles.BottomContent, {
+            className={classNames(Styles.DisputeTabs, {
```

Comment:
this is a common component shouldn't have any references to impl `DisputeTabs` naming doesn't belong here